Source/Core/AudioCommon/AudioCommon.cpp

```diff
@@ -65,14 +65,18 @@ namespace AudioCommon
 		if (g_sound_stream)
 		{
 			UpdateSoundStream();
-			if (g_sound_stream->Start())
+			if (!g_sound_stream->Start())
 			{
-				if (SConfig::GetInstance().m_DumpAudio && !s_audio_dump_start)
-					StartAudioDump();
-
-				return g_sound_stream;
+				PanicAlertT("Could not initialize backend %s. Will use the Null backend instead.", backend.c_str());
```

Comment:
I'm not sure if this should be PanicAlert or ERROR_LOG. As we catch the error, it's not fatal, but for the user, not having sound is likely almost so.